examples/cra-kitchen-sink/.storybook/config.js

```diff
@@ -20,3 +20,5 @@ function loadStories() {
 }
 
 configure(loadStories, module);
+
+export { getStorybook };
```

Comment:
That seems great, although can you explain a bit more why? Should this be part of this PR? It sounds like a seperate thing